Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -253,7 +253,7 @@ void NetworkStorageManager::close(CompletionHandler<void()>&& completionHandler)
         connection.removeWorkQueueMessageReceiver(Messages::NetworkStorageManager::messageReceiverName());
     });
 
-    m_queue->dispatch([this, protectedThis = Ref { *this }, completionHandler = WTFMove(completionHandler)]() mutable {
+    m_queue->dispatch([this, protectedThis = Ref { *this }, completionHandler = WTFMove(completionHandler), storageManager = WTFMove(m_sharedServiceWorkerStorageManager)]() mutable {
```

Comment:
m_sharedServiceWorkerStorageManager is set in the work queue, so it could be moved here, but set later on to a non null value.
It might be good to:
- Add a WTF_GUARDED_BY_CAPABILITY to m_sharedServiceWorkerStorageManager
- If setting m_sharedServiceWorkerStorageManager to null in the lambda does not work, std::exchange should work here.

Reply:
Good point. Fixed.